global/settings/types/preferencekeys.h

```diff
@@ -112,6 +112,7 @@
 #define PREF_UI_CANVAS_FG_USECOLOR_IN_PALETTES              "ui/canvas/foreground/useColorInPalettes"
 #define PREF_UI_CANVAS_BG_COLOR                             "ui/canvas/background/color"
 #define PREF_UI_CANVAS_FG_COLOR                             "ui/canvas/foreground/color"
+#define PREF_UI_CANVAS_FG_EXPORT_PDF                        "ui/canvas/foreground/export/pdf"
```

Comment:
Isn't that rather "background"?

Reply:
And why restrict this to PDF and not having it for PNG, SVG, and possibly MusicXML too?

Reply:
I don't think that it is "background". The background color is the one behind the page.

Reply:
As far as MusicXML is concerned, I will probably make some decently sized changes in the MusicXML code, so I'll add png and svg for now.

Reply:
forgroung color is that of notes, staff lines etc, (black by default), bachound is white by default and your PR allows to change that. And not just for PDF anymore now, it seems

Reply:
![image](https://user-images.githubusercontent.com/21060365/79209695-709a5c00-7e4c-11ea-9b56-62719adc7c89.png)

The background color is dark-blue by default and the paper color is white. 

![image](https://user-images.githubusercontent.com/21060365/79209784-87d94980-7e4c-11ea-9b42-8773752c1ceb.png)

The paper color is called fgColor in prefsdialog.*

You are probably right, but I seem unable to understand my mistake. I am essentially copying the implementation of use the same color in palettes (I should change the name of my macro).